vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,17 +104,30 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
+    private List<String> extractValues(IExcelSheetParser sheetParser, String range)
+    {
+        return sheetParser.getDataFromRange(range).stream().map(CellValue::getValue).collect(Collectors.toList());
+    }
+
     private List<String> extractDataFromAddresses(IExcelSheetParser sheetParser, String addresses)
     {
         return Stream.of(addresses.split(";")).map(sheetParser::getDataFromCell).collect(Collectors.toList());
     }
 
-    private String build(List<String> data, TableProperties properties)
+    private String build(List<String> headers, List<List<String>> data, TableProperties properties)
+    {
+        String lineBreakReplacementPropertyValue = properties.getProperties().getProperty("lineBreakReplacement");
+        String lineBreakReplacement = lineBreakReplacementPropertyValue == null ? ""
+            : lineBreakReplacementPropertyValue;
+        List<List<String>> result = data.stream().map(element -> replaceLineBreaks(element, lineBreakReplacement)).
+                                    collect(Collectors.toList());
+        return ExamplesTableProcessor.buildExamplesTableFromColumns(headers, result, properties);
+    }
+
+    private List<String> replaceLineBreaks(List<String> list, String lineBreakReplacement)
     {
-        String columnName = ExtendedTableTransformer.getMandatoryNonBlankProperty(properties, "column");
-        String joinValues = properties.getProperties().getProperty("joinValues");
-        List<String> columnData = Boolean.parseBoolean(joinValues) ? List.of(String.join(" ", data)) : data;
-        return ExamplesTableProcessor.buildExamplesTableFromColumns(List.of(columnName),
-                List.of(columnData), properties);
+        return list.stream()
+                     .map(e -> e.replace("\n", lineBreakReplacement))
```

Comment:
are there any guarantees that "\n" only Unix-like linebreaks will be used?

Reply:
I think that's OOS the current PR, I have no idea how it worked before, but we don't have issues